### PR TITLE
Allow to pass units to diff method

### PR DESCRIPTION
**Pull request by andriyor**

### CODE REVIEW

File: packages/core/IUtils.d.ts [hunk 1]
```diff
@@ -59,6 +59,17 @@ export interface DateIOFormats<TLibFormatToken = string> {
   keyboardDateTime24h: TLibFormatToken;
 }
 
+export type Unit =
```
Comment (dmtrKovalenko) [inline packages/core/IUtils.d.ts:62]: Form of the names better to be in the plural form. “years”, “seconds”, etc

File: packages/dayjs/src/dayjs-utils.ts [hunk 2]
```diff
@@ -129,7 +129,7 @@ export default class DayjsUtils implements IUtils<defaultDayjs.Dayjs> {
     return date === null;
   }
 
-  public getDiff(date: Dayjs, comparing: Dayjs, units?: any, float?: any) {
+  public getDiff(date: Dayjs, comparing: Dayjs, units?: Unit, float?: any) {
```
Comment (dmtrKovalenko) [inline packages/dayjs/src/dayjs-utils.ts:132]: What float means here?

**Review by dmtrKovalenko — CHANGES_REQUESTED** [review]
Fix CI

File: packages/core/IUtils.d.ts (hunk 1 above)
Comment (andriyor) [reply packages/core/IUtils.d.ts:62]: just added plural forms of units

File: packages/dayjs/src/dayjs-utils.ts (hunk 2 above)
Comment (andriyor) [reply packages/dayjs/src/dayjs-utils.ts]: that was earlier
just deleted this

**Comment by andriyor** [comment]
> Fix CI

sorry but i don't know how to fix such error

**Review by dmtrKovalenko — APPROVED** [review]

File: packages/date-fns-jalali/src/date-fns-jalali-utils.ts [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
-import differenceInMilliseconds from "date-fns-jalali/differenceInMilliseconds";
+import differenceInYears from "date-fns/differenceInYears";
+import differenceInQuarters from "date-fns/differenceInQuarters";
+import differenceInMonths from "date-fns/differenceInMonths";
+import differenceInWeeks from "date-fns/differenceInWeeks";
+import differenceInDays from "date-fns/differenceInDays";
+import differenceInHours from "date-fns/differenceInHours";
+import differenceInMinutes from "date-fns/differenceInMinutes";
+import differenceInSeconds from "date-fns/differenceInSeconds";
+import differenceInMilliseconds from "date-fns/differenceInMilliseconds";
```
Comment (smmoosavi) [inline packages/date-fns-jalali/src/date-fns-jalali-utils.ts:12]: these function should be imported from `date-fns-jalali`

I just fixed theme in #550

**Pull request merged**